src/test/javascript/spec/component/team/team-participation-table.component.spec.ts

```diff
@@ -161,11 +161,11 @@ describe('TeamParticipationTableComponent', () => {
 
     let router: any;
 
-    beforeEach(async () => {
+    beforeEach(() => {
         router = new MockRouter();
         return TestBed.configureTestingModule({
-            imports: [ArtemisTestModule, ArtemisDataTableModule, MockModule(NgxDatatableModule), MockModule(RouterModule), MockModule(NgJhipsterModule), MockModule(NgbModule)],
-            declarations: [TeamParticipationTableComponent, MockPipe(ArtemisTranslatePipe), MockPipe(ArtemisDatePipe)],
+            imports: [ArtemisTestModule, MockModule(NgxDatatableModule), MockModule(RouterModule), MockModule(NgJhipsterModule), MockModule(NgbModule)],
```

Comment:
clone-repo-button.component.spec.ts mocks the NgbModule, this file imports it. Do we know which one is faster/preferrable?

Reply:
It is probably better to mock the module. I don't know the performance difference because I didn't do detailed measurements, but seeing how importing the shared module and a mock of the shared module took both a long time I don't think mocking alone will give you that many performance benefits during the import.